scripts/linux/electron-installer-appimage.sh

```diff
@@ -105,7 +105,6 @@ fi
 if [ -z "$TMPDIR" ]; then
   TMPDIR=$(mktemp -d)
```

Comment:
Probably need to change this script to pass in `TEMPORARY_DIRECTORY` and then remove this if-block too?